composer.json

```diff
@@ -16,16 +16,20 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.3.3",
+        "psr/http-message": "~1.0",
+        "symfony/http-foundation": "~2.3"
     },
     "require-dev": {
+        "symfony/phpunit-bridge": "~2.7",
+        "zendframework/zend-diactoros": "~1.0"
```

Comment:
If we wanted to support PHP 5.3, it would be better to remove the `zendframework/zend-diactoros` package here and install it only on Travis when PHP is 5.5 or higher. This way, we would be able to test our real lowest allowed dependency versions.